daemon/src/maker_inc_connections.rs

```diff
@@ -370,6 +370,10 @@ impl Actor {
 
             async move {
                 while let Ok(()) = this.send(SendHeartbeat(identity)).await {
+                    tracing::debug!(
+                        "Sent heartbeat message. Sleeping for {}",
+                        heartbeat_interval.as_secs()
+                    );
                     tokio::time::sleep(heartbeat_interval).await;
                 }
             }
```

Comment:
I'd suggest to add a trace after the loop to see if the maker ever runs into scenarios where the maker fails to send a heartbeat to the taker and stops. This might be useful to see as well.

Reply:
great idea, added!

Reply:
I added it as a `warn`, as this should not happen during normal operation.